docs/windows.md

```diff
@@ -0,0 +1,187 @@
+```
+Bcdedit.exe -set TESTSIGNING ON
+Restart-Computer
+```
+
+Then, install the OVS using script provided by antrea.
+```
+curl.exe -LO https://raw.githubusercontent.com/ruicao93/antrea/windows_install/hack/windows/Install-OVS.ps1
+.\Install-OVS.ps1
+```
+Verify the OVS services are installed.
+```
+ovs-vsctl show
+```
+
+2.Install wins, kubelet, kubeadm and configure kubelet startup params.
+
+Firstly, install wins, kubelet, kubeadm using script `PrepareNode.ps1` provided by kubernetes.
+```
+Example:
+curl.exe -LO https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/PrepareNode.ps1
+.\PrepareNode.ps1 -KubernetesVersion v1.17.0
+```
+Then, set the Node IP used by kubelet.
+Open file `/var/lib/kubelet/kubeadm-flags.env`:
+```
+KUBELET_KUBEADM_ARGS="--cgroup-driver= --network-plugin=cni --pod-infra-container-image=k8s.gcr.io/pause:3.1"
+```
+Append "--node-ip=$NODE_IP" arg at the end of params. Replace `$NODE_IP` with the address for kubelet. It should look like:
+```
+KUBELET_KUBEADM_ARGS="--cgroup-driver= --network-plugin=cni --pod-infra-container-image=k8s.gcr.io/pause:3.1 --node-ip=$NODE_IP"```
+```
+$cmd = "C:\k\kubelet.exe $global:KubeletArgs --cert-dir=$env:SYSTEMDRIVE\var\lib\kubelet\pki --config=/var/lib/kubelet/config.yaml --bootstrap-kubeconfig=/etc/kubernetes/bootstrap-kubelet.conf --kubeconfig=/etc/kubernetes/kubelet.conf --hostname-override=$(hostname) --pod-infra-container-image=`"mcr.microsoft.com/k8s/core/pause:1.3.0`" --enable-debugging-handlers --cgroups-per-qos=false --enforce-node-allocatable=`"`" --network-plugin=cni --resolv-conf=`"`" --log-dir=/var/log/kubelet --logtostderr=false --image-pull-progress-deadline=20m --node-ip=$env:NODE_IP"
+```
+
+3.Prepare network adapter for kubernetes service.
+Kube-proxy need a network adapter to configure kubernetes service IPs and use the adapter for network proxy.
+```
+curl.exe -LO https://raw.githubusercontent.com/ruicao93/antrea/windows_install/hack/windows/Prepare-ServiceInterface.ps1
+.\Prepare-ServiceInterface.ps1
+```
+
+3.Run kubeadm to join the node.
+
+On windows worker node, use the command that was given to you when you ran `kubeadm init` on a control plane host to join the node to cluster.
+
+If you no longer have this command, or the token has expired, you can run `kubeadm token create --print-join-command` (on a control plane host) to generate a new token and join command.
+```
+Example:
+kubeadm join 192.168.101.5:6443 --token tdp0jt.rshv3uobkuoobb4v     --discovery-token-ca-cert-hash sha256:84a163e57bf470f18565e44eaa2a657bed4da9748b441e9643ac856a274a30b9
+```
+
+#### Verify your installtion
+There will be temporary network interruption on windows worker node on the first startup of Antrea Agent. It's because Antrea Agent will set the OVS to take over the host network. Then you should now be able to view the Windows nodes and Pods in your cluster by running:
```

Comment:
s/Then you should now be able to view/After that you should be able to view

Reply:
Done